.circleci/config.yml

```diff
@@ -15,6 +15,7 @@ jobs:
       - run:
           name: Run pre-commit tests
           command: pre-commit run --all-files
+      - run: curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.21.0
```

Comment:
I'm wondering if we should just stick this thing in the circleci-primary-image.

Reply:
If this is something that'll be used across CircleCI projects (i.e., repos), then it makes sense to put it on our image. How big is it?

If this is a one off, then I'd look for a way to cache this install rather than having the build do this every time.

Reply:
We basically use it on all our Go project at this point.

Reply:
Should I make a task to put in our image and to pull out from repos that have this? (I can go through and look unless someone has a mental list).

Reply:
Yeah, that sounds good.

Reply:
https://www.pivotaltracker.com/story/show/169218329